cmd/tools/microservice/copy/environment.go

```diff
@@ -0,0 +1,80 @@
+package copy
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"os"
+	"os/exec"
+
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var environmentCMD = &cobra.Command{
+	Use:   "environment",
+	Short: "Copy env variables configmap for a microservice in one environment to another environment",
+	Long: `
+	Copy env variables configmap for a microservice in one environment to another environment.
+
+	go run main.go tools studio cfg copyenv --application cde2e951-d40a-3548-8b45-64c0ded97940 \
+		--microservice-name frontend \
+		--from-env test \
+		--to-env prod
+	`,
```

Comment:
```suggestion
	go run main.go tools microservice copy environment --application cde2e951-d40a-3548-8b45-64c0ded97940 \
		--microservice-name frontend \
		--from-env test \
		--to-env prod
	`,
```

Just fixing the old doc string

Reply:
Agreed!

Reply:
Fixed.